docs/source/gbd2017_models/concept_models/vivarium_swissre_breastcancer/concept_model.rst

```diff
@@ -253,44 +253,38 @@ Scale-up of breast cancer screening coverage among insured population
-|            | Southwest | 7%     |                                             |
-+------------+-----------+--------+                                             |
-| Henan      | Central   | 17%    |                                             |
-+------------+-----------+--------+                                             |
-| Helilong-  | Northeast | 8%     |                                             |
-| jiang      +-----------+--------+                                             |
-|            | Northwest | 8%     |                                             |
-+------------+-----------+--------+---------------------------------------------+                                                                           
 
-.. todo::
- currently adds up to 101%
++---------------------------------------------------------------------------------------------------+
+| Population size weight table                                                                      | 
++============+=============+========+===============+===============================================+
+| Province   | location_id | Weight | Weighted ACMR | Forecasted ACMR in log space                  |
++------------+-------------+--------+---------------+-----------------------------------------------+
+| Tianjian   |  517        | 18%    | e^(mr) x 0.18 | filepath                                      |
++------------+-------------+--------+---------------+ :download:`asmr<filepaths_c429_forecast.xlsx>`|                                             
+| Jiangsu    |  506        | 28%    | e^(mr) x 0.28 |                                               |
++------------+-------------+--------+---------------+ Note: GBD does not produce estimates below    |
+| Guangdong  |  496        | 22%    | e^(mr) x 0.22 | province level, so we do not have data for    |
++------------+-------------+--------+---------------+ sub-provinces. Therefore, we are summing      |
+| Henan      |  502        | 16%    | e^(mr) x 0.16 | the sub-province weights to get total         |
++------------+-------------+--------+---------------+ province weights for Guangdong and            |
+| Heilong-   |  501        | 16%    | e^(mr) x 0.16 | Heilongjiang.                                 |
+| jiang      |             |        |               |                                               |
++------------+-------------+--------+---------------+-----------------------------------------------+
 
+.. note::
 
-.. _5.3:
+  The forecasted data in the .nc files are in log space. The acmr is labelled as 'mr'. To get the simulation population's all cause mortality rate (acmr), first take the exponential of the mr values for location, in the .nc files, then mulitply by the population weight, and sum over all locations. The unit after the log transformation is in person years. Multiple by 100,000 to get per 100,000 person years.                                                                            
```

Comment:
the unit after _exp_ transform (not log)

Reply:
Probably we can specify the original ACMR estimate (column_name = 'mr')  is in log space with base of natural e to avoid the distraction. Besides that, the data transformation steps totally make sense to me.

Reply:
thanks, good description, I added :)